common/statistics/src/clients/mod.rs

```diff
@@ -36,9 +36,9 @@ impl ClientStatsSender {
     /// Report a statistics event using the sender.
     pub fn report(&self, event: ClientStatsEvents) {
         if let Some(tx) = &self.stats_tx {
-            if let Err(err) = tx.send(event) {
-                log::error!("Failed to send stats event: {:?}", err);
-            }
+            tx.send(event)
+                .inspect_err(|err| log::debug!("Failed to send stats event: {err}"))
```

Comment:
nit-question: any reason for downgrading it from `error` to `debug`?

Reply:
Mostly because it proved difficult to pass a TaskClient all the way down here and sending stats didn't seem critical (compared to other tasks)

But I'll see if I can maybe change that by moving the TaskClient from the run call arguments to the task structs instead